api/src/main/java/edu/cornell/mannlib/vitro/webapp/web/templatemodels/individual/BaseIndividualTemplateModel.java

```diff
@@ -104,7 +104,7 @@ public String getRdfUrl() {
         }
     }
 
-    public GroupedPropertyList getPropertyList() {
+    public GroupedPropertyList getPropertyList() throws Exception {
```

Comment:
Not a good idea to add a generic exception here. Should at least be limited to throwing the exceptions that occur in the method, but also consider whether they can be handled within the method and not changing the signature.